worlds/messenger/__init__.py

```diff
@@ -167,7 +170,11 @@ def fill_slot_data(self) -> Dict[str, Any]:
         }
 
     def get_filler_item_name(self) -> str:
-        return "Time Shard"
+        if not getattr(self, "_filler_items", None):
+            self._filler_items = [name for name in self.random.choices(list(FILLER),
+                                                                       weights=list(FILLER.values()),
+                                                                       k=100)]
```

Comment:
```suggestion
            self._filler_items = self.random.choices(list(FILLER), weights=list(FILLER.values()), k=20)
```

Reply:
i guess it's better to have a smaller cache, and need to create it more frequently, than just create a too large one once?

Reply:
in my experience, probably yes. I did not benchmark it.